code/modules/mob/living/simple_animal/abnormality/he/road_home.dm

```diff
@@ -140,7 +140,7 @@
 		return
 
 	playsound(src, 'sound/abnormalities/roadhome/House_NormalAtk.ogg', 100, FALSE, 8)
-	for(var/mob/living/L in livinginrange(15, src))
+	for(var/mob/living/L in livinginview(15, src))
```

Comment:
Please use In View, Living in view takes all the mobs on the Z level gets their distance to target, and should only really be used about 50

Reply:
We should probably change rudolta then, fixed.